backend/app/views/spree/admin/orders/index.html.erb

```diff
@@ -52,13 +52,21 @@
           <div class="row">
             <div class="col-12 col-xl-6">
               <div class="field">
+<<<<<<< HEAD
                 <%= label_tag :q_number_cont, Spree.t(:order_number, number: '') %>
                 <%= f.text_field :number_cont %>
+||||||| parent of e13e004... Switch order search to using starts over cont
+                <%= label_tag :q_number_cont, Spree.t(:order_number, :number => '') %>
+                <%= f.text_field :number_cont %>
+=======
+                <%= label_tag :q_number_start, Spree.t(:order_number, :number => '') %>
+                <%= f.text_field :number_start %>
+>>>>>>> e13e004... Switch order search to using starts over cont
```

Comment:
@gmacdougall I think this has been introduced rebasing and fixing the CHANGELOG conflict.